gatsby-node.js

```diff
@@ -128,17 +128,17 @@ exports.createPages = ({ graphql, actions }) => {
         }
 
         const tagSet = new Set();
-        const categorySet = new Set();
+        // const categorySet = new Set();
         result.data.allMarkdownRemark.edges.forEach(edge => {
           if (edge.node.frontmatter.tags) {
             edge.node.frontmatter.tags.forEach(tag => {
               tagSet.add(tag);
             });
           }
 
-          if (edge.node.frontmatter.category) {
```

Comment:
Moving forward, let's make sure we remove unused code instead of commenting. I'll wipe this in my next PR so no need to address now.